src/transport/AdminPairingTable.cpp

```diff
@@ -281,6 +339,12 @@ CHIP_ERROR AdminPairingInfo::GetCredentials(OperationalCredentialSet & credentia
         certificates.LoadCert(mRootCert, mRootCertLen,
                               BitFlags<CertDecodeFlags>(CertDecodeFlags::kIsTrustAnchor).Set(CertDecodeFlags::kGenerateTBSHash)));
 
+    if (mICACert != nullptr && mICACertLen > 0)
```

Comment:
Comment on the purpose of this block.

Reply:
It optionally adds the whole certificate chain (including ICA) if ICA exists.

Reply:
I meant to say "Add a comment on the purpose of this block" :) We reuse the `mICACert != nullptr && mICACertLen > 0` condition a lot. Perhaps it would be good to have `IsICACAvailable` and `IsNOCAvailable` trivial bool const predicates to reduce the mental burden of these pointer/size checks in the code.